.github/tsc.json

```
@@ -4,14 +4,15 @@
             "owner": "tsc",
             "pattern": [
                 {
-                    "regexp": "^(?:\\s+\\d+\\>)?([^\\s].*)\\((\\d+|\\d+,\\d+|\\d+,\\d+,\\d+,\\d+)\\)\\s*:\\s+(error|warning|info)\\s+(\\w{1,2}\\d+)\\s*:\\s*(.*)$",
+                    "regexp": "^(?:\\s+\\d+\\>)?([^\\s].*)\\((\\d+)(?:,(\\d+)(?:,\\d+(?:,\\d+)?)?)?\\)\\s*:\\s+(error|warning|info)\\s+(\\w{1,2}\\d+)\\s*:\\s*(.*)$",
```

```suggestion
                    "regexp": "^([^\\s].*)[\\(:](\\d+)[,:](\\d+)(?:\\):\\s+|\\s+-\\s+)(error|warning|info)\\s+TS(\\d+)\\s*:\\s*(.*)$",
```

(Official regex from vscode at https://github.com/microsoft/vscode/blob/cd41dd1f6065db76c77c6e2f659bb50e6f558dec/extensions/typescript-language-features/package.json#L974-L981 )